tests/tests_create_lvmvdo_then_remove.yml

```diff
@@ -0,0 +1,118 @@
+---
+- hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    mount_location: '/opt/test1'
+    volume_group_size: '10g'
+    volume_size: '12g'
+    pool_size: '9g'
+
+  tasks:
+    - include_role:
+        name: linux-system-roles.storage
+
+    - name: gather package facts
+      package_facts:
+        # gather information about packages
+
+    - name: set name of blivet package
+      set_fact:
+        blivet_pkg_name: "{{ ansible_facts.packages | select('search','blivet') | select('search', 'python') | list }}"
+
+    - name: set blivet package version
+      set_fact:
+        blivet_pkg_version: "{{ ansible_facts.packages[blivet_pkg_name[0]][0]['version'] + '-' + ansible_facts.packages[blivet_pkg_name[0]][0]['release'] }}"
+
+    - block:
+      - include_tasks: get_unused_disk.yml
```

Comment:
the tasks inside the block need to be indented by an additional two spaces - so
```yaml
        - block:
            - include_tasks: get_unused_disk.yml
              vars:
...
```

Reply:
Thanks for noticing. Fixed.